src/main/java/org/victorrobotics/dtlib/command/Command.java

```diff
@@ -44,7 +44,7 @@ default void execute() {}
    * @return whether the command has finished
    */
   default boolean isFinished() {
-    return true;
+    return false;
```

Comment:
Is this the same as or different from the wpilib default? Also, does this affect how isFinished needs to behave for instant commands, run commands, or any other command types?

Reply:
All built-in commands override the default, so no additional implementation changes are needed. Additionally, the WPILib default is the same as the new version (don't know why I didn't just do it this way originally)